WORKING_GROUPS.md

```diff
@@ -99,7 +99,7 @@ The i18n working groups handle more than just translations. They
 are endpoints for community members to collaborate with each
 other in their language of choice.
 
-Each team is organized around a common spoken language. Each
+Each team is organized around a commonly spoken language. Each
```

Comment:
Isn't there a significant difference in meaning of:
`a common spoken language`
and
`a commonly spoken language`